src/Popover/components/ContentWrapper.js

```diff
@@ -180,7 +191,8 @@ const PopoverContentWrapper = ({
   });
   const verticalPosition = calculateVerticalPosition(position[0], dimensions);
   const horizontalPosition = calculateHorizontalPosition(position[1], dimensions);
-  const actionsDimensions = useMemo(() => boundingClientRect(actionsRef), []);
+  const actionsDimensions = useMemo(() => boundingClientRect(actionsRef), [actionsRef.current]);
```

Comment:
Ive adde your suggestion, which is correct, however I also had to add a setTimeout, due to content having animation thus causing it to not calculate properly if you have idea how to get rid of the setTimeout and not cause another re-render as it was in useMemo case then let me know 🤔 

Also the best option for this might be to implement [callback ref](https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node) but this would not handle recalculation on actions change

Reply:
Wow, callback ref is a much nicer way to solve this problem! 😃 The dependency for `useCallback` should then be `[actions]`, and use the same timeout to account for the animation. 💪 The delay is no problem because 15 ms is unnoticeable.

One more suggestion: instead of saving the entire bounding rect object, you can only save height to state because that's the only value you're using. Also, setting initial value for the bounding rect to `0` is my mistake, but if you're going to change it to contain only height, then `0` is good. 👍